scripts/smp-server-digitalocean-droplet/scripts/90-cleanup.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
I still didn't get why we version these scripts, let's discuss offline

Reply:
I think the point of having them here is that we can reliably rebuild the snapshot in case some of their files change…